distribution/helm/vector-shared/templates/_labels.tpl

```diff
@@ -9,7 +9,12 @@ helm.sh/chart: {{ include "libvector.chart" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
+app.kubernetes.io/component: logging
```

Comment:
@jszwedko I'm curious what you think about this - since we advertise Vector as a the one-stop-shop for logs/metrics/(soon to be events)

Reply:
@spencergilbert `app.kubernetes.io/component: observability` ?

Reply:
After talking to @jszwedko about this label, we'd rather punt on it and just let users set it themselves with the new `commonLabels`/`customLabels` key